e2e/main_page.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test';
+
+test('has correct title', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page).toHaveTitle('Angular Cookbook Example');
+});
+
+test('has login as admin button', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page.locator(`[data-testid='login-admin__button']`)).toBeVisible();
+});
+
+test('has login as employee button', async ({ page }) => {
+  await page.goto('/');
+
+  await expect(page.locator(`[data-testid='login-employee__button']`)).toBeVisible();
+});
```

Comment:
As suggested in the other comments, we can avoid tests for checking elements on the screen that won't be hidden otherwise in any condition. Which means if we're at the home page, those buttons are there already.
We should however test what happens when we click them. Which you do in the tests below 👍🏽